All changes related to GA 04/21 (#95)
Kong/insomnia-docs: docs/insomnia/git-sync.md

![Enable Git Sync for Documents by clicking the Setup Git Sync button inside a Document.](/assets/images/document-git-sync.png)
_Inside a Design Document, click Setup Git Sync._

> ### Sign-out of Git Account

Reviewer: ```suggestion
### Sign out of Git account
```

> Signout of synced Git Management Account.  Click on **Setup Git Sync**, then click on **Repository Settings**.  In the box there is a list of all synced accounts.  Next to each account is a **Sign Out** button.  Click on **Sign Out** for the accounts that need to be signed out.  Another box will appear asking for confirmation.  Click on **Sign Out** again.  Then the **Congifgure Respoitory** box will no longer have the account that was signed out in its list of configured accounts.

Reviewer: ```suggestion
Sign out of a synced Git management account.  

1. Click on **Setup Git Sync**, then click on **Repository Settings**.  
2. You should see a list of all synced accounts. Next to each account is a **Sign Out** button. Click on **Sign Out** for the accounts that need to be signed out.  
3. In the box that appears, click **Sign Out** again.  

The **Configure Repository** box will no longer list the account that was signed out in its list of configured accounts.
```

## Remote Repository Settings

When configuring a remote repository, you can chose to either connect with GitHub or manually setup a remote repository

### Setup a remote repository with GitHub:

If you are not authorised click the 'Authenticate with GitHub' link.
You will be prompted to connect your GitHub account with Insomnia.
Once the authorisation flow is finished you will be navigated back to Insomnia.

> You can now clone any repository from GitHub!

Reviewer: Where exactly do they click this link? How do they get there?

> If you are not authorised click the 'Authenticate with GitHub' link.
> You will be prompted to connect your GitHub account with Insomnia.
> Once the authorisation flow is finished you will be navigated back to Insomnia.
> You can now clone any repository from GitHub!

Reviewer: ```suggestion
Click the **Authenticate with GitHub** link. You will be prompted to connect 
your GitHub account with Insomnia.

When the authorization flow is finished, you will be navigated back to Insomnia.
You can now clone any repository from GitHub!
```

> ### Manually setup a remote repository

Reviewer: ```suggestion
### Manually set up a remote repository
```

![Enable Git Sync for Documents by clicking the Setup Git Sync button inside a Document.](/assets/images/document-git-sync.png)
_Inside a Design Document, click Setup Git Sync._

> ### Sign out of Git account

Reviewer 2: This should be moved down until after you have figured out how to sign in.

3. In the box that appears, click **Sign Out** again.

The **Configure Repository** box will no longer list the account that was signed out in its list of configured accounts.

## Remote Repository Settings

When configuring a remote repository, you can chose to either connect with GitHub or manually set up a remote repository.

### Set up a remote repository with GitHub

1. Click on the **Setup Git Sync** on the upper right hand corner of the Insomnia App.  A tab will appear under the box.
2.  Click **Repository Settings**.  This will open a box called **Configure Respository**
3. Click on the button **Authenticate with Github**. A tab will open in your default browser that says "Connect with Github".
4. Click on **Continue**. If the browser has already been authenticated with Github, the page will say "Successfully authenticated Insomnia".
5.  You might be prompted to continue by your browser through the Insomnia app via  "Choose Application" box.  If you are not, you can follow the instructions on the page to complete Github account authentication with the Insomnia App.

> 6. You can now clone any repository from GitHub!

Reviewer 2: We probably need a little more here. Like:
> 6. You can now clone any repository from GitHub! Just copy and paste an `https` URI for the GitHub repository you want to connect to in the "GitHub URI" input box.

Verbiage to be improved ^

Reviewer: Adjusting Wil's phrasing a little:

> 6. You can now clone any repository from GitHub! Copy the HTTPS URI for the GitHub repository you want to connect to and paste it into the "GitHub URI" field.
